pyproject.toml

```diff
@@ -55,7 +55,8 @@ dependencies = [
     # ensure that we always use versions that do not have",
     # known security vulnerabilities",
     "tornado>=6.3.3",
-    "ipython>=8.10.0",
+    "ipython>=8.20; python_version >= '3.10'",
+    "ipython==8.10; python_version == '3.9'",
```

Comment:
I think it should be fine to leave this as is we can use 8.10 and up for all python versions